stubs/colorama/colorama/ansitowin32.pyi

```diff
@@ -0,0 +1,50 @@
+import sys
+from typing import Any, Callable, Dict, Optional, Pattern, Sequence, TextIO, Tuple, Union
+from typing_extensions import Protocol
+
+if sys.platform == "win32":
+    from .winterm import WinTerm
+
+    winterm: WinTerm
+else:
+    winterm: None
+
+class SupportsWrite(Protocol):
```

Comment:
You could use `from _typeshed import SupportsWrite`

Reply:
done